tests/testsuite_basic/data/cockpit-ostree/packit.yaml

```diff
@@ -3,6 +3,7 @@ synced_files:
   - cockpit-ostree.spec
 upstream_project_name: cockpit-ostree
 downstream_package_name: cockpit-ostree
+upstream_project_url: "https://github.com/cockpit-project/cockpit-ostree"
```

Comment:
Could I edit those files or I need to upstream that change first?

Reply:
I'm fine with this being downstream first if you don't forget to submit it upstream later :-)